currency/rates_test.go

```diff
@@ -50,33 +51,39 @@ func TestUnMarshallRates(t *testing.T) {
 			expectedError: nil,
 		},
 		{
-			desc: "Valid JSON field defines a conversions map with repeated entries, expect error",
+			desc: "Valid JSON field defines a conversions map with repeated entries, last one wins",
 			ratesJSON: `{
 				"conversions":{
 					"USD":{
 						"GBP":0.7662523901,
 						"MXN":20.00
 					},
 					"USD":{
-						"GBP":0.7662523901
-					},
+						"GBP":0.4815162342
+					}
```

Comment:
The behavior of json and json-iter match here. The test was incorrectly setup. It failed because of the trailing comma, nothing to do with the repeated elements. Fixed the test and manually verified behavior matches.